Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:10-buster as build-deps
+LABEL maintainer="Penn Labs"
+
+ARG GHOST_API_KEY
+ENV GHOST_API_KEY ${GHOST_API_KEY}
```

Comment:
Does this actually work? Peyton and I were having issues with it

Reply:
worked for me testing locally

Reply:
yeah this works i  have tested it out